config/initializers/carrier_wave.rb

```diff
@@ -0,0 +1,18 @@
+CarrierWave.configure do |config|
+  if Rails.env.staging? || Rails.env.production?
+    config.storage    = :aws
+    config.aws_bucket = ENV.fetch('S3_BUCKET_NAME')
+    config.aws_acl    = :'public-read'
+    config.asset_host = 'http://example.com'
+    config.aws_authenticated_url_expiration = 60 * 60 * 24 * 365
+
+    config.aws_credentials = {
+      access_key_id:     ENV.fetch('AWS_ACCESS_KEY_ID'),
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.